source/guide_ttrss.rst

```diff
@@ -0,0 +1,200 @@
+.. highlight:: console
+
+.. author:: Tobias Bell <uberspace@tobiasbell.de>
+
+.. sidebar:: About
+
+  .. image:: _static/images/ttrss.png
+      :align: center
+
+##############
+Tiny Tiny RSS
+##############
+
+Tiny Tiny RSS is a free and open source web-based news feed (RSS/Atom) reader and
+aggregator, designed to allow you to read news from any location, while feeling as
+close to a real desktop application as possible.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * MySQL_
+  * supervisord_
+  * domains_
+  * git_
+  * PHP_
+  * cron_
+
+License
+=======
+
+The software is licensed under the GPL. All relevant information can be found in the repository of the project. 
+
+  * https://git.tt-rss.org/git/tt-rss
+
+Prerequisites
+=============
+
+We're using PHP in the latest stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+The URL where you want to run Tiny Tiny RSS needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Clone the Tiny Tiny RSS repository
+----------------------------------
+
+As Tiny Tiny RSS uses a rolling release model with the git master branch as current stable version, we
+wil use git to download the software. So go to the webroot and clone Tiny Tiny RSS into the subfolder tt-rss
+
+::
+
+ [isabell@stardust ~]$ cd ~/html
+ [isabell@stardust ~]$ git clone https://tt-rss.org/git/tt-rss.git tt-rss
+ Klone nach 'tt-rss' ...
+ remote: Counting objects: 59813, done.
+ remote: Compressing objects: 100% (23687/23687), done.
+ remote: Total 59813 (delta 32579), reused 59789 (delta 32557)
+ Empfange Objekte: 100% (59813/59813), 74.56 MiB | 1.28 MiB/s, Fertig.
+ Löse Unterschiede auf: 100% (32579/32579), Fertig.
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Setup your database
+-------------------
+
+It's a good idea to create the tables under a separate database.
+
+::
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE isabell_tt_rss"
+
+Replace <isabell> with your own user. 
+
+Configure database and domain
+-----------------------------
+
+Now you should reach your installation under the URL ``https://<username>.uber.space/tt-rss`` (would be ``https://isabell.uber.space/tt-rss`` in our example).
+Open the URL and set the Database settings to
+
+ * Database Type: MySQL
```

Comment:
In other guides that's: "your MySQL hostname, username and password: the hostname is localhost and you should know your MySQL credentials by now. If you don’t, start reading again at the top." Can you change that for consistency?